config/core/test-configs.yaml

```diff
@@ -2327,6 +2327,21 @@ test_configs:
   - device_type: r8a774a1-hihope-rzg2m-ex
     test_plans:
       - baseline
+      - baseline-nfs
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-lkdtm
+      - kselftest-seccomp
+      - ltp-crypto
+      - ltp-fcntl-locktests
+      - ltp-ima
+      - ltp-ipc
+      - ltp-mm
+      - ltp-pty
+      - ltp-timers
+      - preempt-rt
+      - smc
```

Comment:
I don't see it on staging for this device, all the results appear to be running on QEMU:
https://staging.kernelci.org/test/job/kernelci/branch/staging-next/kernel/staging-next-20211020.0/plan/smc/

Reply:
And here is smc results https://lava.ciplatform.org/scheduler/job/522573

Reply:
That's not a test job from staging.kernelci.org.

Reply:
OK: https://staging.kernelci.org/test/plan/id/6184961ba299bc6c43a6c425/

With one test case failure, but I think it's a common issue.